specific_guidelines.Rmd

```diff
@@ -46,31 +46,31 @@ Translate:
 
 3. comments;
 
-4. user function names and parameters names: if a code snippet creates a function as example, we can name it and their parameters in English. 
+4. user function names and parameters names: if a code snippet creates a function as example, we can name the function and its parameters in English. 
 
 > example:
     
     Original code:
     
     adentro <- function(punto, mas_bajo, mas_alto) {
             if (punto <= mas_bajo) {
-                return false
+                return(FALSE)
```

Comment:
I know it's example code but it has to be R code to not hurt my eyes :stuck_out_tongue_winking_eye: